### ISSUE TITLE
clicking an element inside the 'target' throws exception.

### ISSUE THREAD

**Issue by pawlufelice** [zenorocha/clipboard.js#119]
I have a button with a span in it. The Clipboard instance is created with the button as the target, but clicking on the span throws an error (listener checks if target has required attributes). The onClick should use `e.currentTarget` instead of `e.target` I think:

```
    onClick(e) {
        if (this.clipboardAction) {
            this.clipboardAction = null;
        }

        this.clipboardAction = new ClipboardAction({
            action  : this.action(e.currentTarget),
            target  : this.target(e.currentTarget),
            text    : this.text(e.currentTarget),
            trigger : e.currentTarget,
            emitter : this
        });
    }
```
Thanks,
Paul

**Comment by pawlufelice** [comment]
http://jsfiddle.net/9v22s40p/

**Comment by louisameline** [comment]
:+1:

**Comment by deronsizemore** [comment]
+1 for me on this issue

**Comment by lwm** [comment]
:rocket:

**Comment by m90** [comment]
:+1:

**Comment by zenorocha** [comment]
Fixed on [v1.5.4](https://github.com/zenorocha/clipboard.js/releases/tag/v1.5.4)! Thanks @pawlufelice for contributing!

**Comment by zenorocha** [comment]
And let me know if this new release works for you ;)
